PlanMyTrip/PlanMyTrip/SelectServicesViewController.m

```diff
@@ -1,13 +1,22 @@
 //
 //  SelectServicesViewController.m
-//  
+//  PlanMyTrip
 //
 //  Created by Farida Abdelmoneum on 7/6/22.
 //
 
 #import "SelectServicesViewController.h"
 
 @interface SelectServicesViewController ()
+@property (nonatomic) BOOL checkedH;
+@property (nonatomic) BOOL checkedF;
+@property (nonatomic) BOOL checkedC;
+@property (weak, nonatomic) IBOutlet UIButton *carsB;
+@property (weak, nonatomic) IBOutlet UIButton *flightsB;
+@property (strong, nonatomic) IBOutlet UIButton *hotelsB;
```

Comment:
Let's name them properly with meaning. this helps other engineer to understand your code.
```
BOOL isHotelSelected
BOOL isFlightSelected
BOOL isCarSelected

UIButton *carsButton
UIButton *flightsButton
UIButton *hotelsButton
```